types/requests.go

```diff
@@ -7,3 +7,16 @@ type ScaleServiceRequest struct {
 	ServiceName string `json:"serviceName"`
 	Replicas    uint64 `json:"replicas"`
 }
+
+// InfoRequest provides information about the underlying provider
+type InfoRequest struct {
+	Provider      string          `json:"provider"`
+	Version       ProviderVersion `json:"version"`
+	Orchestration string          `json:"orchestration"`
+}
+
+// ProviderVersion provides the commit sha and release version number of the underlying provider
+type ProviderVersion struct {
+	Sha     string `json:"sha"`
```

Comment:
I think Go would have us name this `SHA` in the struct and keep the json notation of `sha`